src/SAWScript/Prover/What4.hs

```diff
@@ -176,11 +191,13 @@ proveWhat4_solver :: forall st t ff.
   Set VarIndex       {- ^ Uninterpreted functions -} ->
   SharedContext      {- ^ Context for working with terms -} ->
   Prop               {- ^ A proposition to be proved/checked. -} ->
+  IO ()              {- ^ Extra setup actions -} ->
   IO (Maybe CEX, SolverStats)
   -- ^ (example/counter-example, solver statistics)
-proveWhat4_solver solver sym unintSet sc goal =
+proveWhat4_solver solver sym unintSet sc goal extraSetup =
   do
      (argNames, bvs, lit, stats) <- setupWhat4_solver solver sym unintSet sc goal
+     _ <- extraSetup
```

Comment:
Style nitpick: the `_ <-` part is unnecessary since `extraSetup` returns `IO ()`:

```suggestion
     extraSetup
```